lib/Pipeline/Context.cpp

```diff
@@ -20,3 +20,42 @@ Logger<> pipeline::CommandLogger("commands");
 
 Context::Context() : TheKindRegistry(Registry::registerAllKinds()) {
 }
+
+llvm::Error Context::store(const revng::DirectoryPath &Path) const {
+  if (auto Error = Globals.store(Path); Error)
+    return Error;
+
+  revng::FilePath IndexPath = Path.getFile(".index");
+  auto MaybeWritableFile = IndexPath.getWritableFile();
+  if (not MaybeWritableFile)
+    return MaybeWritableFile.takeError();
+
+  MaybeWritableFile->get()->os() << GlobalIndex;
```

Comment:
Newline?

Reply:
It can be added, but then it requires `.trim`-ing at deserialization

Reply:
OK. I don't like text files that do not end with `\n` `:D`

Reply:
Done